pipelines/batch/README.md

```diff
@@ -0,0 +1,193 @@
+
+### JDBC input parameters
+
+JDBC mode is used if `jdbcModeEnabled=true`.
+
+To use JDBC mode, first create a copy of
+[hapi-postgres-config.json](https://github.com/google/fhir-data-pipes/blob/master/utils/hapi-postgres-config.json)
+and edit the values to match your database server.
+
+Next, include the following parameters:
+
+- `jdbcModeEnabled=true`
+- `fhirDatabaseConfigPath=./path/to/config.json`
+
+If you are using a HAPI FHIR server, also include:
+
+- `jdbcModeHapi=true`
+- `jdbcDriverClass=org.postgresql.Driver`
+
+All JDBC parameters:
+
+- `jdbcModeEnabled` - If true, uses JDBC mode. Default: `false`
+- `fhirDatabaseConfigPath` - Path to the FHIR database config for JDBC mode.
+  Default: `../utils/hapi-postgres-config.json`
+- `jdbcModeHapi` - If true (with `jdbcModeEnabled`), uses JDBC mode for a HAPI
+  source server. Default: `false`
+- `jdbcFetchSize` - The fetch size of each JDBC database query. Default: `10000`
+- `jdbcMaxPoolSize` - The maximum number of database connections. Default: `50`
+- `jdbcDriverClass` - The JDBC driver to use. Should be set to
+  `org.postgresql.Driver` for HAPI FHIR Postgres access. Default:
+  `com.mysql.cj.jdbc.Driver`
+
+### Parquet output parameters
+
+Parquet files are output when `outputParquetPath` is set.
+
+- `outputParquetPath` - The file path to write Parquet files to, e.g.,
+  `./tmp/parquet/`. Default: empty string, which does not output Parquet files.
+- `secondsToFlushParquetFiles` - The number of seconds to wait before flushing
+  all Parquet writers with non-empty content to files. Use `0` to disable.
+  Default: `3600`.
+- `rowGroupSizeForParquetFiles` - The approximate size in bytes of the
+  row-groups in Parquet files. When this size is reached, the content is flushed
+  to disk. This is not used if there are less than 100 records. Use `0` to use
+  the default Parquet row-group size. Default: `0`.
+
+### FHIR output parameters
+
+Resources will be copied to the FHIR server specified in `fhirSinkPath` if that
+field is set.
+
+- `fhirSinkPath` - A base URL to a target FHIR server, or the relative path of a
+  GCP FHIR store, e.g. `http://localhost:8091/fhir` for a FHIR server or
+  `projects/PROJECT/locations/LOCATION/datasets/DATASET/fhirStores/FHIR-STORE-NAME`
+  for a GCP FHIR store. If using a GCP FHIR store,
+  [see here for setup information](https://github.com/google/fhir-data-pipes/wiki/Create-a-Google-Cloud-FHIR-Store-and-BigQuery-Dataset).
+  default: none, resources are not copied
+- `sinkUserName` - The HTTP Basic Auth username to access the FHIR sink. Not
+  used for GCP FHIR stores.
+- `sinkPassword` - The HTTP Basic Auth password to access the FHIR sink. Not
+  used for GCP FHIR stores.
+
+### A note about Beam runners
+
+If the pipeline is run on a single machine (i.e., not on a distributed cluster),
+for large datasets consider using a production grade runner like Flink. This can
+be done by adding the parameter `--runner=FlinkRunner` (use `--maxParallelism`
```

Comment:
For `FlinkRunner` option, consider adding a link to its documentation [here](https://beam.apache.org/documentation/runners/flink/).